Middle click will close tab

voy opened the pull request.

voy commented on lib/Tab.js:
```diff
@@ -52,7 +54,6 @@ var Tab = React.createFactory(React.createClass({
 	},
 	dragOut:function(){
-		console.log("out");
 		delete this.state.draggingOver;
```
state should be manipulated via `this.setState`

VanCoding commented:
Looks good! :)
But there are 3 spots that are indented by spaces instead of tabs, could you maybe fix that before I merge?

VanCoding commented on lib/Tab.js:
```diff
@@ -30,8 +30,10 @@ var Tab = React.createFactory(React.createClass({
-	select:function(e){
-		if(e.nativeEvent.shiftKey || e.nativeEvent.ctrlKey){
+	click:function(e){
+        if(e.button === 1) {
```
tabs

VanCoding commented on lib/TabManager.js:
```diff
@@ -12,23 +12,23 @@ var TabManager = React.createFactory(React.createClass({
+					selection:this.state.selection,
+					hiddenTabs:this.state.hiddenTabs,
+					filterTabs:this.state.filterTabs,
+                    tabMiddleClick:this.deleteTab.bind(this),
```
tabs

VanCoding commented on lib/TabManager.js:
```diff
@@ -94,9 +92,11 @@ var TabManager = React.createFactory(React.createClass({
 			});
 		}
 	},
+    deleteTab:function(tabId) {
```
tabs

voy commented:
Of course. My bad, sorry!

voy commented:
Fixed tabs, added `.editorconfig` to hopefully prevent such issues in the future. Sorry about the delay, but I have good excuse – my second son was born in January :-).

VanCoding commented:
Congrats, @voy =)

It looks good now, I'll merge it tomorrow.
I've never dealt with a .editorconfig file btw, does it also work with atom?

voy commented:
There definitely are plugins for most if not all modern editors, [atom included](https://atom.io/packages/editorconfig) :-).

voy commented:
yay!
